procedures/unit-testing-basics.ipf

```diff
@@ -3447,6 +3453,13 @@ Function RunTest(procWinList, [name, testCase, enableJU, enableTAP, enableRegExp
 
 	else
 		// no early return/abort above this point
+		if(!!fixLogName)
```

Comment:
Please use a separate line before to properly initialize fixLogName.
Like:
`fixLogName = ParamIsDefault(fixLogName) ? 0 : !!fixLogName`

This way it is easy to see what the default value of the optional parameter should be.
Then please also add that value to the fixLogName argument comment description above. (default 0 disabled)